lib/quandl/model/list.rb

```diff
@@ -0,0 +1,44 @@
+module Quandl
+  class List
+    attr_reader :meta
+    attr_reader :values
+
+    def initialize(klass, values, meta)
+      @klass = klass
+      @values = values.map { |v| klass.new(v, meta: meta) }
+      @meta = meta
+    end
+
+    def more_results?
+      @meta['total_pages'] > @meta['current_page']
```

Comment:
test for all 3 models now since, expected behaviour of Data is to raise an exception